### PR TITLE
Adds C1-TA DEL-B radio, supposedly fixes "That's Life" being a pop song

### PR DESCRIPTION
**Pull request by Funkeronipasta**
## About The Pull Request

Adds multiple VA-11 Hall-A tracks under their own category.
Moves "That's Life" by Frank Sinatra from "Pop" to "Jazz and Lounge"

## Why It's Good For The Game

VA-11 Hall-A tracks tie into the C1-TA DEL-B bar theme. Adds further tracks and reduces the risk of eardrum-breaking Arcade tracks suddenly playing by being in its own category.

Frank Sinatra's "That's Life" should not be in Pop, makes it hard as hell to find when all the rest are in Jazz and Lounge.

## Changelog
:cl:
add: Garoad's "Welcome to VA-11 Hall-A" under "Arcade"
add: Garoad's "Every Day is Night" under "Arcade"
add: Garoad's "Commencing Simulation" under "Arcade"
add: Garoad's "Good for Health, Bad for Education" under "Arcade"
add: Garoad's "JC Elton's" under "Arcade"
add: Garoad's "Snowfall" under "Arcade"
add: Garoad's "Last Call" under "Arcade"
add: Garoad's "Karmotrine Dream" under "Arcade"
add: Garoad's "Strictly Business" under "Arcade"
add: Garoad's "Dusk" under "Arcade"
add: Garoad's "Meet the Staff" under "Arcade"
add: Garoad's "Underground Club" under "Arcade"
tweak: Frank Sinatra's "That's Life" genre "Pop" changed to "Jazz and Lounge"
/:cl:

### CODE REVIEW

**Comment by drofoljaelisglis** [comment]
For one, I don't know if one selection of songs deserves an entire genre slot dedicated to them. That'd be like calling an album it's own genre. It would fit more seamlessly and be less of an obvious reference if all the tracks were moved to their own category. If you think Arcade is a poor fit, why not move them to Electronic?

For two, it's obvious you didn't test your code beyond compilation to see if it works, as testing with the code you have currently will simply break the tracklist, in the sense that no songs in any genre will display. Nor will the new genre, in fact. I'm not **certain** if the indentation makes a difference for .json files, so that may or may not be an issue - something for you test when you go back and re-do the PR. There's other errors beyond that though. Read through the code again carefully, and if it still isn't working, then go and compare your code versus the last iteration.

**Comment by Funkeronipasta** [comment]
Thank you for the input. I'm not familiar with ss13 code, so I was simply working with what I could figure out. I'll shift the genres and have a look at the indentation, but there wasn't an issue on my end (that may just be Visual Studio being itself). I wasn't able to test if it worked due to Dream Daemon crashing before server start. At least one other user has reported having the same issue, so I assumed it was a BYOND thing since the compile came back clean.

File: code/game/machinery/jukebox.json [hunk 1]
```diff
@@ -101,16 +91,7 @@
...
-"url": "https://files.catbox.moe/rgtjn3.m4a",
-"title": "Welcome to VA-11 Hall-A",
-"duration": 1880,
-"artist": "Garoad",
-"secret": false,
-"lobby": false,
-"jukebox": true,
-"genre": "Arcade"
-},
+
```
Comment (nik707) [inline code/game/machinery/jukebox.json:94]: what

**Comment by Captain277** [comment]
Closing this one due to a collection of conflicts and the amount of time it's gone untouched. Feel free to update the branch and I'll re-open it. Or you can resubmit it!

**Pull request closed without merging**